### ISSUE TITLE
Slow uploading speed

### ISSUE THREAD

**Issue by MuhammadAqib1122** [blackcodetavern/ESP32_FTPClient#10]
Hi, I am using ESP32 development board to read files from the sd card and send them to the FTP server using this library but the speed is terribly slow. It is taking about 10mins to upload 10mb file.

Is there any way to increase the speed?

**Comment by ldab** [comment]
Have you tested your FTP server with a Windows client for example and compared the speed?

**Comment by MuhammadAqib1122** [comment]
Yes. From windows client, speed is good. But from esp32, speed is very slow.
